doc/api/util.md

```diff
@@ -220,13 +220,15 @@ as a `printf`-like format string which can contain zero or more format
 specifiers. Each specifier is replaced with the converted value from the
 corresponding argument. Supported specifiers are:
 
-* `%s` - `String` will be used to convert all values except `BigInt` and
-  `Object`. `BigInt` values will be represented with an `n` and Objects are
-  inspected using `util.inspect()` with options
-  `{ depth: 0, colors: false, compact: 3 }`.
-* `%d` - `Number` will be used to convert all values except `BigInt`.
-* `%i` - `parseInt(value, 10)` is used for all values except `BigInt`.
-* `%f` - `parseFloat(value)` is used for all values.
+* `%s` - `String` will be used to convert all values except `BigInt`, `Object`
+  and `-0`. `BigInt` values will be represented with an `n` and Objects that
+  have no own `toString` function are inspected using `util.inspect()` with
+  options `{ depth: 0, colors: false, compact: 3 }`.
+* `%d` - `Number` will be used to convert all values except `BigInt`, `Symbol`
+  and `-0`.
```

Comment:
These documentation changes seem separate.

Reply:
I updated the whole documentation while being on it. Shall I move the `-0` changes to a separate commit?

Reply:
That's what I had meant, yeah.

Reply:
I separated the change into an individual commit. PTAL (I also removed the `-0` part from e.g. `%d` because `Number` will still be used to convert the entry, it will just be represented by -0 instead of 0 as it was in older versions).